mypy/config_parser.py

```diff
@@ -287,6 +284,22 @@ def _find_config_file(
 
     return None
 
+def parse_and_validate_filenames(
```

Comment:
When a function has an `and` in the name, this usually indicates that it's doing too many things. Although, validation is usually implied, so I'd drop its mention from the name. Additionally, it's possible to name it more accurately:
```suggestion
def convert_raw_files_string_to_list(
```

Reply:
It may be reasonable to make it private too:
```suggestion
def _convert_raw_files_string_to_list(
```